.circleci/update_coverage_description.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+
+report_type=$1
+
+if [[ -f htmlcov/index.html ]]; then
+    pr_number=${CIRCLE_PULL_REQUEST##*/}
+    body=$(gh pr view | sed -e '1,13d')
+    coverage_label=$(grep -oP '(?<=<span class="pc_cov">).+(?=<\/span>)' htmlcov/index.html)
+
+    if [[ "$body" == *"${report_type}:"* ]]; then
+        body=$(echo "$body" | sed -e "s/${report_type}: [.*](https:\/\/.*\/index.html)/${report_type}: [${coverage_label}](https:\/\/output.circle-artifacts.com\/output\/job\/${CIRCLE_WORKFLOW_JOB_ID}\/artifacts\/0\/tmp\/coverage\/htmlcov-${report_type}\/index.html)/g")
```

Comment:
I would move the replacing string with the artifact url into a variable with a name that's easier to understand, and then reuse it in the sed command.  (it looks like they are identical)

e.g., 
`ci_report_artifact="https:\/\/output.circle-artifacts.com\/output\/job\/${CIRCLE_WORKFLOW_JOB_ID}\/artifacts\/0\/tmp\/coverage\/htmlcov-${report_type}\/index.html)"`

Reply:
Great idea.